code/modules/recycling/disposal.dm

```diff
@@ -368,7 +368,6 @@
 		if( contents.len )
 			if(mode == 2)
 				spawn(0)
-					feedback_inc("disposal_auto_flush",1)
```

Comment:
?

Reply:
Why the hell were we logging these is my first question